src/main/java/seedu/kitchenhelper/parser/Parser.java

```diff
@@ -134,6 +138,44 @@ public Command prepareAddRecipe(String attributes) throws KitchenHelperException
         return addCmd;
     }
 
+    /**
+     * Checker for command format for addrecipe command.
+     *
+     * @param command   The full user input without keyword.
+     * @return  true if is of valid command, otherwise false.
+     */
+    public Boolean checkAddRecipeCmdFormat(String command) {
```

Comment:
Good fix for the addRecipe command with regards to the ghost parameters and missing invalid parameters!